lib/action_subscriber/middleware/env.rb

```diff
@@ -40,9 +41,9 @@ def initialize(subscriber, encoded_payload, properties)
         @subscriber = subscriber
       end
 
-      def acknowledge
+      def acknowledge(acknowledge_multiple_messages = nil)
```

Comment:
Seems like you can make this `acknowledge_multiple_messages = false` and remove the check below.